admin/orders.php

```diff
@@ -352,16 +352,16 @@ function couponpopupWindow(url) {
             <div class="form-group col-xs-4 col-sm-3 col-md-3 col-lg-3">
                 <?php
                 echo zen_draw_form('search', FILENAME_ORDERS, '', 'get', '', true);
-                echo zen_draw_label(HEADING_TITLE_SEARCH_ALL, 'search', 'class="sr-only"');
+                echo zen_draw_label(HEADING_TITLE_SEARCH_ALL, 'searchAll', 'class="sr-only"');
```

Comment:
Shouldn't this stay "search"? as it refers to the name of the input field?

Reply:
zen_draw_label make a **Label** with a **For** pointing at an input with id=search.
That does not (currently) exist so the For is invalid.
If you make the corresponding input have a corresponding id=search, that duplicates the name of the form, so not valid.
Hence I used a new value searchAll for the Label-For input and used that as the id in the corresponding input.
When the form is submitted, the value of the input is still passed using the existing name value of **search** as before, so it still works.
Having a changed id would affect only css and javascript things, not the form.